olm-catalog/serverless-operator/project.yaml

```diff
@@ -68,7 +68,7 @@ dependencies:
     nodejs_20_minimal: registry.access.redhat.com/ubi8/nodejs-20-minimal@sha256:a2a7e399aaf09a48c28f40820da16709b62aee6f2bc703116b9345fab5830861
     openjdk_21: registry.access.redhat.com/ubi8/openjdk-21@sha256:441897a1f691c7d4b3a67bb3e0fea83e18352214264cb383fd057bbbd5ed863c
     python-39: registry.access.redhat.com/ubi8/python-39@sha256:27e795fd6b1b77de70d1dc73a65e4c790650748a9cfda138fdbd194b3d6eea3d
-  operator: 1.15.4
+  operator: 1.16.1
```

Comment:
Should we go with [v1.16.2](https://github.com/knative/operator/releases/tag/knative-v1.16.2)? I thas a change that might be useful to Eventing folks.

Reply:
The commit is there since 1.16.0 if you refer to this one: https://github.com/knative/operator/commit/83d44d7da394f1b2c203cda50c9535b060dc5fdc.

The release notes are copied from original 1.16 release. Anyway I can update the version to be on the latest one for sure.

Reply:
Updated